ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -229,6 +243,19 @@ applyPatchHeapClosures = do
             "reallyUnsafePtrEqualityUpToTag# :: Any -> Any -> Int#\nreallyUnsafePtrEqualityUpToTag# _ _ = 0#\n"
         =<< readFile' file
 
+-- | Fix up these rts include paths. We don't ship rts headers - we go
```

Comment:
```suggestion
-- | Fix up these rts include paths. We don't ship rts headers since we run ghc-lib using the RTS of the compiler we build with - we go
```